packages/universal-sql/src/client-duckdb/browser.js

```diff
@@ -42,24 +42,42 @@ export async function initDB() {
 	await db.open({ query: { castBigIntToDouble: true, castTimestampToDate: true } });
 }
 
+/**
+ * Updates the duckdb search path to include only the list of included schemas
+ * @param {string[]} schemas
+ * @returns {Promise<void>}
+ */
+export async function updateSearchPath(schemas) {
```

Comment:
This and `node.js`' definitions could be merged into something in `both.js` with an extra `db` param and re-exported here (also applies to `setParquetURLs`

Reply:
It looks like there are some interop conflicts happening here, going to leave as-is for now